skins/fluent2/QskFluent2Skin.cpp

```diff
@@ -1587,7 +1587,7 @@ void Editor::setupTextInputColors(
 
     for( const auto state : { A::NoState, Q::Hovered, Q::Focused, Q::Editing, Q::Disabled } )
     {
-        QRgb panelColor, borderColor1, borderColor2, textColor;
+        QRgb panelColor = Qt::black, borderColor1 = Qt::black, borderColor2 = Qt::black, textColor = Qt::black;
 
         if ( state == A::NoState )
```

Comment:
`state` was 0x4000 in my scenario, don't know how and why but nonetheless the variables should be initialized

Reply:
I think we should rather convert the "if( state == A::NoState)" clause to an else clause that catches all other cases.

Qt::black doesn't work here because we should always use a color from the palette due to the day/night themes (black couldn't be read in the night theme).

Reply:
done